src/test/java/com/github/dockerjava/client/AbstractDockerClientTest.java

```diff
@@ -38,7 +41,7 @@
 
     public static final Logger LOG = LoggerFactory.getLogger(AbstractDockerClientTest.class);
 
-    private String apiVersion = "1.19";
+    private String apiVersion = "1.21";
```

Comment:
Seems thats not that simple. `RemoteApiVersion` is not used by `DockerClientConfigBuilder` so far. So leaving this for separate PR.